code/modules/mob/living/simple_animal/friendly/spiderbot.dm

```diff
@@ -200,8 +200,9 @@
 	if(camera)
 		camera.status = 0
 
-	held_item.loc = src.loc
-	held_item = null
+	if (held_item): // if the spiderbot is holding an item
```

Comment:
``:``
Please remove or justify

Reply:
I apology. Coding in Python is slowling taking over my motor functions.